### PR TITLE
Migrate themes to dart-sass module system

### PR DESCRIPTION
**Pull request by epetrow**
closes: https://github.com/telerik/kendo-themes-private/issues/270
closes: https://github.com/telerik/kendo-themes-private/issues/271
closes: https://github.com/telerik/kendo-themes-private/issues/272
closes: https://github.com/telerik/kendo-themes-private/issues/273
closes: https://github.com/telerik/kendo-themes-private/issues/274
closes: https://github.com/telerik/kendo-themes-private/issues/275

### CODE REVIEW

File: .github/workflows/_compile-themes.yml [hunk 1]
```diff
@@ -37,23 +37,21 @@ jobs:
 
       - name: Build test assets
         run: |
-          npm run sass
+          npm run sass --prefix packages/core
+          npm run sass --prefix packages/default
+          npm run sass --prefix packages/utils
```
Comment (Juveniel) [inline .github/workflows/_compile-themes.yml:42]: we should update this once we merge all themes in the branch

File: .github/workflows/_create-screenshots.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        theme: [default, bootstrap, material, classic, nouvelle, fluent]
+        theme: [default]
```
Comment (Juveniel) [inline .github/workflows/_create-screenshots.yml:17]: same here

File: .github/workflows/_lint-styles.yml [hunk 3]
```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        theme: [default, bootstrap, material, classic, nouvelle, fluent]
+        theme: [default]
```
Comment (Juveniel) [inline .github/workflows/_lint-styles.yml:18]: same here

File: packages/default/README.md [hunk 4]
```diff
@@ -65,21 +67,10 @@ We have more than 2000 variables, so it's hard to list them all. We've tried to
...
-
-// When you are finished with your customization, import the theme
-@import "@progress/kendo-theme-default/dist/all.scss";
-
-// Note: the variables bellow this point will not take effect,
-// because the theme is already imported
-
-$grid-border: red;
-
+@use "@progress/kendo-theme-default/scss/all/scss" as * with (
```
Comment (Juveniel) [inline packages/default/README.md:70]:
```suggestion
@use "@progress/kendo-theme-default/scss/all.scss"
```

**Review by Juveniel — APPROVED** [review]

**Review by silviyaboteva — APPROVED** [review]
Massive work! Great output! 🚀

File: packages/default/scss/index.scss [hunk 5]
```diff
@@ -1,181 +1,338 @@
...
-@import "./color-preview/_index.scss";
-@import "./loader/_index.scss";
-// @import "./loading/_index.scss";
-@import "./skeleton/_index.scss";
-@import "./time-marker/_index.scss";
+@forward "./avatar/_index.scss";
+@forward "./badge/_index.scss";
+@forward "./color-preview/_index.scss";
+@forward "./loader/_index.scss";
+// @use "./loading/_index.scss";
```
Comment (inikolova) [inline packages/default/scss/index.scss:29]: We could remove this comment

File: packages/default/scss/index.scss [hunk 6]
```diff
@@ -1,181 +1,338 @@
...
+@use "./no-data/_index.scss" as *;
+
+
+// Indicators
+@use "./avatar/_index.scss" as *;
+@use "./badge/_index.scss" as *;
+@use "./chip/_index.scss" as *;
+@use "./color-preview/_index.scss" as *;
+@use "./loader/_index.scss" as *;
+// @use "./loading/_index.scss" as *;
```
Comment (inikolova) [inline packages/default/scss/index.scss:197]: We could remove this comment

File: packages/core/scss/border-radii/index.import.scss [hunk 7]
```diff
@@ -1,27 +1,29 @@
+@use "sass:map";
+
 /// The none border radius used across the Components.
 /// @group radii
-$kendo-border-radius-none: k-map-get($kendo-spacing, 0) !default;
+$kendo-border-radius-none: 0px !default;
```
Comment (inikolova) [inline packages/core/scss/border-radii/index.import.scss:5]: Could we keep the spacing mapping, instead of hardcode values?

**Pull request merged**